### ISSUE TITLE
Show support for First-Party-Cookie attribute

### ISSUE THREAD

**Issue by ericlaw1979** [MicrosoftEdge/Status#201]
In Chrome, and a great idea.
https://tools.ietf.org/html/draft-west-first-party-cookies-01
https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/wKZBCzcNssg/GxwA-Y8Gs84J

**Comment by mikewest** [comment]
We're planning on shipping this in Chrome 43ish to see how things go.

**Comment by molant** [comment]
@ericlaw1979, @mikewest can you add the chromestatus id please?

**Comment by mikewest** [comment]
`4672634709082112` (https://www.chromestatus.com/features/4672634709082112)

**Comment by jacobrossi** [comment]
Thanks... I'm just waiting on some details from the networking/storage folks and I'll get this added in.  Hold tight...

**Comment by mikewest** [comment]
Ping! Did you get details from the networking/storage folks? :)

Also, we're up to https://tools.ietf.org/html/draft-west-first-party-cookies-03 now. Still haven't shipped in Chrome, but I'm working on it.

**Comment by mikewest** [comment]
This morphed into `SameSite` cookies, the draft has been adopted by the HTTPWG (https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site), Chrome shipped in 51 (https://www.chromestatus.com/features/4672634709082112), and Mozilla is working on it (https://bugzilla.mozilla.org/show_bug.cgi?id=795346).

Is it something y'all could consider? :)

**Comment by kypflug** [comment]
SameSite cookies shipped in EdgeHTML 17 and serviced downlevel in 1806 - this should be fixed with #616.  @alrra can you confirm and close?

**Comment by ericlaw1979** [comment]
Yes, `SameSite` cookies shipped, but this particular issue was about Cookie Prefixes, a feature which shipped (thanks @mikewest) and [remains in Chrome](https://cs.chromium.org/chromium/src/net/cookies/canonical_cookie.cc?type=cs&q=__SECURE&sq=package:chromium&g=0&l=492). Edge does not map cookie prefixes to SameSite cookies: [Test Page](http://webdbg.com/test/cookie/magicnames.aspx)
